titus-server-master/src/main/java/com/netflix/titus/master/service/management/internal/SimpleCapacityGuaranteeStrategy.java

```diff
@@ -129,7 +129,7 @@ public CapacityAllocations compute(CapacityRequirements capacityRequirements) {
         Map<AgentInstanceGroup, Integer> result = new HashMap<>();
         for (Map.Entry<AgentInstanceGroup, Integer> entry : instanceAllocations.entrySet()) {
             AgentInstanceGroup instanceGroup = entry.getKey();
-            int minSize = instanceGroup.getAutoScaleRule().getMin();
+            int minSize = instanceGroup.getMin();
```

Comment:
Is this valid? We want to enforce some min of min constraint. This was previously set in the auto scale rule. Now it should be 0.

Reply:
Will remove this as we don't need this behavior here anymore.